problems/low-bus-factor.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: low-bus-factor
+stages:
+  - development
+short_description: Low bus factor is a risk for team work when one or more important team members get stuck (from various reasons) and cannot work. 
+tags:
+  - Low bus factor
+  - team
+  - redundancy
+  - fresh documentation
+  - automation
+keywords:
+  - bus
+  - factor
+  - team
+  - methodology
+---
+
+# Low Bus Factor
+
+## TL;DR
+
+How many people need to get hit by a bus for the project or team to stop functioning? What are the ways how to increase this number?  
+
+![](/files/bus1.png)
```

Comment:
Please, add source.
```suggestion
![](/files/bus1.png)
[example](url)
```

Reply:
I added the figure source into Resources, but OK – added also under the figure.